content/tutorials/get-started/get-started-using-python.md

```diff
@@ -0,0 +1,215 @@
+{% set n = cycler(* range(1,99)) %}
+
+When you're working with the XRP Ledger, there are a few things you'll need to manage, whether you're adding XRP into your [wallet](wallets.html), integrating with the [decentralized exchange](decentralized-exchange.html), or [issuing tokens](issued-currencies.html). This tutorial walks you through basic patterns common to getting started with all of these use cases and provides sample code for implementing them. 
+
+Here are the basic steps you'll need to cover for almost any XRP Ledger project:
+
+1. [Connect to the XRP Ledger.](#connect)
+1. [Generate a wallet.](#generate-wallet)
+1. [Query the XRP Ledger.](#query)
+
+
+### {{n.next()}}. Connect
+
+To make queries and submit transactions, you need to establish a connection to the XRP Ledger. To do this with `xrpl-py`, use the [`xrp.clients` module](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.clients.html):
+
+
+{{ include_code("_code-samples/xrpl-py/get-acct-info.py", start_with="# Define the network client", end_before="# Create a wallet using the testnet faucet:", language="py") }}
+
+
+### {{n.next()}}. Generate wallet
+
+To store value and execute transactions on the XRP Ledger, you need to create a wallet: a [set of keys](cryptographic-keys.html#key-components) and an [address](accounts.html#addresses) that's been [funded with enough XRP](accounts.html#creating-accounts) to meet the [account reserve](reserves.html#reserves). The address is the identifier of your account and you use the [private key](cryptographic-keys.html#private-key) to sign transactions that you submit to the XRP Ledger. 
+
+
+For testing and development purposes, you can use the [XRP Faucets](xrp-testnet-faucet.html) to generate keys and fund the account on the Testnet or Devnet. For production purposes, you should take care to store your keys and set up a [secure signing method](set-up-secure-signing.html). 
+
+
+To make it easy to create a wallet on the Testnet, `xrpl-py` provides the [`generate_faucet_wallet`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.wallet.html#xrpl.wallet.generate_faucet_wallet) method:
+
+
+
+{{ include_code("_code-samples/xrpl-py/get-acct-info.py", start_with="# Create a wallet using the testnet faucet:", end_before="# Create an account str from the wallet", language="py") }}
+
+
+This method returns a [`Wallet` instance](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.wallet.html#xrpl.wallet.Wallet):
+
+
+```py
+print(test_wallet)
+
+# print output
+seed: shrPSF6vV3v3yoND9J3NeKks6M3xd
+pub_key: 022FA613294CD13FFEA759D0185007DBE763331910509EF8F1635B4F84FA08AEE3
+priv_key: 00D46D49A1A260C24964E6CFD5C85308ACE958E200C14B4DCA73AA5C87288C8F6C
+classic_address: raaFKKmgf6CRZttTVABeTcsqzRQ51bNR6Q
+```
+
+#### Using the wallet
+
+In this tutorial we only query details about the generated account from the XRP Ledger, but you can use the values in the `Wallet` instance to prepare, sign, and submit transactions with `xrpl-py`.   
```

Comment:
This process will likely be modified in a couple of future PRs (including PR 197)

Reply:
Looks likehttps://github.com/XRPLF/xrpl-py/pull/197 isn't merged yet, so I'll leave this as-is for now and will update this after the merge happens.

Reply:
Updated these per that PR and a few other changes.